icu4j/main/classes/core/src/com/ibm/icu/text/UnicodeSet.java

```diff
@@ -144,6 +144,13 @@
  * their delimiters; "[:^foo]" and "\P{foo}".  In any other location,
  * '^' has no special meaning.
  *
+ * <p>Since ICU 70, "[^...]", "[:^foo]", "\P{foo}", and "[:binaryProperty=No:]"
+ * perform a “code point complement” (all code points minus the original set),
+ * removing all multicharacter strings,
+ * equivalent to .{@link #complement()}.{@link #removeAllStrings()} .
```

Comment:
Question: what does this look like when Javadoc ​renders it?

Optional: Make the code samples more prominent by putting it in its own `<pre>`, rather than inline in the paragraph.  (Maybe in C++, too)

Reply:
Each `{@link <method>}` renders in code style with a link to the JavaDoc of the method. Eclipse auto-complete helped write these links.

For this simple sequence of two function calls I think this is fine.